.dockerignore

```diff
@@ -4,6 +4,14 @@ client_apps/canvas_quizzes/node_modules/
 client_apps/canvas_quizzes/dist/
 client_apps/canvas_quizzes/tmp/
 config/*.example
+config/*.yml
+# Exclude the config files that are actually checked in and should be part of the image
+# Every other config should be set at runtime.
+!config/brandable_css.yml
+!config/browsers.yml
+!config/fontcustom.yml
+!config/styleguide.yml
+!config/testem.yml
```

Comment:
So these files are included in the image?

Reply:
Only the files on lines 10-14 are included in the image. Everything else is excluded. I included those b/c they are actually checked into src ctrl (upstream in the instructure repo) which feels like they aren't really meant to hold anything that really changes. The build was failing without `config/brandable_css.yml` in there so I just went ahead and added all the ones that were checked in.